src/props-invl.c

```diff
@@ -160,25 +174,28 @@ EditIntervalDialog *
 edit_interval_dialog_new (void)
 {
 	EditIntervalDialog *dlg;
-	GladeXML  *glxml;
-	GtkWidget *w, *menu, *menu_item;
+	GladeXML           *glxml;
+	GtkWidget          *w, *menu, *menu_item;
 
-	dlg = g_malloc (sizeof(EditIntervalDialog));
+	dlg = g_malloc (sizeof (EditIntervalDialog));
 	dlg->interval = NULL;
 
 	glxml = gtt_glade_xml_new ("glade/interval_edit.glade", "Interval Edit");
 	dlg->gtxml = glxml;
 
 	dlg->interval_edit = glade_xml_get_widget (glxml, "Interval Edit");
 
-	glade_xml_signal_connect_data (glxml, "on_ok_button_clicked",
-	        GTK_SIGNAL_FUNC (interval_edit_ok_cb), dlg);
+	glade_xml_signal_connect_data (
+	    glxml, "on_ok_button_clicked", GTK_SIGNAL_FUNC (interval_edit_ok_cb), dlg
+	);
 
-	glade_xml_signal_connect_data (glxml, "on_apply_button_clicked",
-	        GTK_SIGNAL_FUNC (interval_edit_apply_cb), dlg);
+	glade_xml_signal_connect_data (
+	    glxml, "on_apply_button_clicked", GTK_SIGNAL_FUNC (interval_edit_apply_cb), dlg
+	);
 
-	glade_xml_signal_connect_data (glxml, "on_cancel_button_clicked",
-	        GTK_SIGNAL_FUNC (interval_edit_cancel_cb), dlg);
+	glade_xml_signal_connect_data (
+	    glxml, "on_cancel_button_clicked", GTK_SIGNAL_FUNC (interval_edit_cancel_cb), dlg
+	);
```

Comment:
I typically end up putting parameters like this (one or several parameters per line), instead
of starting over on a new line:

```c
glade_xml_signal_connect_data (glxml, "on_cancel_button_clicked",
                               GTK_SIGNAL_FUNC (interval_edit_cancel_cb), dlg);
```

That's the style Gtk seems to use in their examples. Although it can get weird sometimes with really long function names.

Reply:
I find this notation a bit odd in the realm of C like languages too. With Ruby it seems to be pretty common. To me it has the two advantages that there are no tab vs. space or whatever issues with indenting the parameters and that a long function name does not push all parameters to the side. I'm not particularly set on this but wouldn't be sad to keep it like this ;-)